projects/packages/videopress/src/client/admin/components/checkbox/index.tsx

```diff
@@ -13,20 +13,28 @@ import styles from './style.module.scss';
 import { CheckboxProps } from './types';
 
 const Checkbox = forwardRef< HTMLInputElement, CheckboxProps >(
-	( { checked, onChange, className, ...inputProps }, ref ) => {
+	( { checked, onChange, className, children, htmlFor, dataTestId, ...inputProps }, ref ) => {
```

Comment:
Just curious, is this `dataTestId` prop used anywhere?

Reply:
It's used in the tests https://github.com/Automattic/jetpack/pull/41853/files#diff-4ba2780cd27b732baa6a7a888cbc0baf2926e3a68c04ddd678f268c19d75fb34R60 😉

Reply:
Sorry! I somehow was not finding it 😄

Reply:
Thanks for surfacing the error. Seems that `VideoRow` and `VideoList` didn't handle the `Checkbox` onChange event correctly. Maybe because `const showCheckbox = false; // TODO: implement bulk actions` 😄. Fixed it now in [6bb9670](https://github.com/Automattic/jetpack/pull/41853/commits/6bb96705a2303a6ac322eb24b7aba98f0b96a6d1)